fdbserver/RestoreMaster.actor.cpp

```diff
@@ -218,14 +225,15 @@ ACTOR static Future<Version> processRestoreRequest(Reference<RestoreMasterData>
                                                    RestoreRequest request) {
 	state std::vector<RestoreFileFR> files;
 	state std::vector<RestoreFileFR> allFiles;
+	state std::map<Version, VersionBatch>::iterator versionBatch = self->versionBatches.end();
```

Comment:
initialize to `end()` seems strange to me. Simply declare it should be fine, since first use will use the `begin()`. Or initialize to `begin()`.

Reply:
Because `kvOps` can be empty, so `begin()`  can be equal to `end()`. To avoid the subtle confusion, I deliberatively used `end()`. I won't mind changing it to `begin()`.

Reply:
I think using `end()` is confusing. I'd prefer to use `begin()` here, or leave uninitialized.

Reply:
Leaving initialized may cause valgrind mistakenly thinking a variable is initialized, I think. 

Since this is not a big deal, I will change it to begin() then.